models/mereology-examples2013/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+title: Example of OntoUML Mereological Relations
+acronym:
+issued: 2013
+modified:
+contributor:
+keyword:
+ - ontouml
+ - mereology
+theme: Class Q - Science
+editorialNote: Ontology presented in a lecture. The ontology was build in a different editor tool and imported to Visual Paradigm, the current diagrammation may not exacly match the original one.
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - learning
```

Comment:
I do not think this applies. See my previous comment about the use of the `learning` task.

Reply:
Done.